src/V3AssertPre.cpp

```diff
@@ -189,6 +257,11 @@ class AssertPreVisitor final : public VNVisitor {
         // Reset defaults
         m_seniDefaultp = nullptr;
     }
+    void visit(AstProperty* nodep) override {
+        // The body will be visited when will be substituted in place of property reference
+        // (AstFuncRef)
+        // This node will be deleted in V3Assert (all references will be substituted then)
```

Comment:
Note you can use `pushDeletep` to delete after the visitor is done.

Reply:
Ok, thanks.